### ISSUE TITLE
[FBX] Importing to .mesh files fails if the FBX contains multiple meshes

### ISSUE THREAD

**Issue by monkeez** [godotengine/godot#50443]
### Godot version

3.3.2.stable

### System information

Windows 10

### Issue description

Importing an FBX mesh in Godot with meshes/storage set to "Files (.mesh)" doesn't keep the names for the meshes. The side effect of this is that it only creates one file called .mesh, so if the FBX has multiple meshes there will be missing .mesh files.

![image](https://user-images.githubusercontent.com/5214414/125560389-54e0c107-7cc3-4f00-9cc2-ab65d2c6be1c.png)

### Steps to reproduce

- Export or download an FBX with multiple meshes (I got mine from Mixamo)
- Create a new Godot project
- Import the FBX with the meshes/storage setting set to Files (.mesh)
- Importer only creates a single mesh file called .mesh

### Minimal reproduction project

[fbx-mesh.zip](https://github.com/godotengine/godot/files/6813435/fbx-mesh.zip)

**Comment by Malcolmnixon** [comment]
I just stumbled across this one attempting to import an asset where all models were inserted into a single FBX file, and I wanted to instance them individually.

I was able to "fix" the issue in Godot 3.4.2 in the EditorSceneImporterFBX::_generate_scene method by adding the following lines:
```
                        // mesh node, mesh id
                        mesh_node = mesh_data_precached->create_fbx_mesh(state, mesh_geometry, fbx_node->fbx_model, p_compress_flags);
                        if (!state.MeshNodes.has(mesh_id)) {
                            state.MeshNodes.insert(mesh_id, fbx_node);
                        }
 +
 +                      // Ensure the mesh has a name for export purposes
 +                      if (mesh_node->get_mesh()->get_name().empty())
 +                          mesh_node->get_mesh()->set_name(fbx_node->node_name);
```

It may have worked in my case, but I'm not sure it's the correct technical solution. It may have been better if create_fbx_mesh could have named the mesh when it was creating it, but unfortunately it's not given the 'fbx_node' object containing the name.

**Comment by akien-mga** [comment]
This is probably fixed with the new FBX importer using [FBX2glTF](https://github.com/V-Sekai/FBX2glTF/releases).
You can test it in Godot 4.0 alpha 6: https://godotengine.org/article/dev-snapshot-godot-4-0-alpha-6
This new importer might also be backported for the `3.x` branch eventually.

If you still find issues with these models using the new importer, we suggest opening a new issue with details about the problems in Godot 4.0 alpha 6 or later.
